appserver/deployment/client/src/main/java/org/glassfish/deployapi/DeploymentFactoryInstaller.java

```diff
@@ -122,29 +123,6 @@ public static DeploymentFactoryInstaller getInstaller() {
         return repository.listFiles();        
     }
     
-    /**
-     * Add a new deployment manager to our respository
-     */
-    public void addDeploymentFactory(File newDM) throws IOException {
-        
-        int number=1;
```

Comment:
is this just dead code removal or intentional ? also is the import change intentional ?

Reply:
Instead of fixing the stream handling in this method, I removed the method after finding that it is not used.

The import is unintentional, my IDE must have made that change. I'll remove it.